pkgs/development/libraries/libucl/default.nix

```
@@ -0,0 +1,29 @@
+{ stdenv, fetchFromGitHub, pkgconfig, autoreconfHook }:
```

Personal preference:

```suggestion
{ stdenv
, ...
, ...
}:
```

---

It makes future updates to this line easier to read / review.